### DISCUSSION TITLE
Deleted organization. Still seen in my profile bug.

### DISCUSSION THREAD

**Discussion by trixmaxzero** [community/community#41002] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

How can i fix this issue ? 

<img width="100%" alt="Screenshot 2022-12-07 at 14 23 41" src="https://user-images.githubusercontent.com/11073291/206168663-97055d23-84f9-462d-bef6-97aad03c1f80.png">
<img width="100%" alt="Screenshot 2022-12-07 at 14 23 56" src="https://user-images.githubusercontent.com/11073291/206168674-d4355472-6b16-4249-bd03-559342137892.png">

**Comment by ossanna16** [comment]
Hi there @thevobos and welcome to our community! Thank you for asking a great question 🙂
While you are waiting for a reply to your question, here are a few things you can do:

To get started, introduce yourself in our [official introduction thread](https://github.com/orgs/community/discussions/23204)
To connect with other new users, check out this Discussion to [Meet Your GitHub Community Team](https://github.com/orgs/community/discussions/28776), further your GitHub knowledge by working through our [GitHub Skill 🆙 exercises](https://github.com/orgs/community/discussions/32121), and participate in our [weekly Fun Friday chats](https://github.com/orgs/community/discussions/30912)
